tests/FSharp.Compiler.Service.Tests/Common.fs

```diff
@@ -475,8 +475,8 @@ let assertRange
 
 [<AutoOpen>]
 module TempDirUtils =
-    let getTempPath dir =
-        Path.Combine(TestFramework.tempDirectoryOfThisTestRun, dir)
+    let getTempPath dir = 
+        Path.Combine(tempDirectoryOfThisTestRun.Value.FullName, dir)
```

Comment:
Not a fan of forcing options, but not such a big deal in tests.

Reply:
Ah, this is a lazy, not an option. I tried to solve the temp dir cleanup somehow in a centralized manner and this is part of the solution.